api/gateway/scramble.go

```diff
@@ -0,0 +1,16 @@
+package gateway
+
+import "github.com/TheThingsNetwork/ttn/utils/scramble"
+
+func (m *GPSMetadata) Scramble() (err error) {
```

Comment:
That's why I added the "A proper delta value should be determined for the gateway location field scrambling. What are your opinions?" point.
How, do you think, this should be done?